docs/docs/models.md

```diff
@@ -841,6 +844,38 @@ To recap&comma; the elements of the order &sol; group array can be the following
   * Everything else is ignored&comma; and if raw is not set&comma; the query will fail
 * Sequelize&period;fn and Sequelize&period;col returns functions and quoted cools
 
+### Indexes
+Sequelize supports adding indexes to the model definition which will be created during `Model.sync()` or `sequelize.sync`.
+
+```js
+sequelize.define('User', {}, {
+  indexes: [
+    // Create a unique index on email
+    {
+      unique: true,
+      fields: ['email']
+    },
+
```

Comment:
Yeah best cover that too, i was a bit too lazy coming up with every single case and figured i'd write some cross cases for demonstration :D